src/main/java/uk/co/evoco/webdriver/configuration/TolerantActionExceptions.java

```diff
@@ -13,7 +13,7 @@
      *
      * @return the tolerant action wait time in seconds
      */
-    public String getWaitTimeoutInSeconds() {
+    String getWaitTimeoutInSeconds() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/gh/digital-delivery-academy/selenium-pom-framework/pullRequest?prid=5208619)

Reply:
changed scoping from package private to public